backend/attachRendererFiber.js

```diff
@@ -181,6 +182,7 @@ function attachRendererFiber(hook: Hook, rid: string, renderer: ReactRenderer):
           if (context && Object.keys(context).length === 0) {
             context = null;
           }
+          memoizedInteractions = publicInstance.memoizedInteractions;
```

Comment:
We should be setting this for the `HostRoot` tag type actually. I can make this change.